### ISSUE TITLE
Move ipynb/cluster_utils.py to a new repo

### ISSUE THREAD

**Issue by josenavas** [biocore/American-Gut#58]
These utilities looks really useful for other project out from the American Gut scope.

Also, removing the IPython dependency will help to make them more general.

**Comment by jwdebelius** [comment]
@josenavas: Has this been done?

**Comment by josenavas** [comment]
I don't think it has been done - but I don't think those utilities are used anymore...
